api/controllers/FlairController.js

```diff
@@ -5,122 +5,65 @@ var _ = require("lodash");
-          css_class = user.flair.ptrades.flair_css_class;
-        } else if (app.sub === "svexchange" && user.flair.svex) {
-          flair = user.flair.svex.flair_text;
-          css_class = user.flair.svex.flair_css_class;
-        }
-        if (app.flair === "involvement" && css_class && css_class.indexOf("1") === -1) {
-          app.flair = user.flair.ptrades.flair_css_class + "1";
-        } else if (app.flair === "involvement") {
-          app.flair = user.flair.ptrades.flair_css_class;
-        } else if (css_class && css_class.indexOf("1") > -1) {
-          app.flair += "1";
-        }
-        if (css_class && css_class.slice(-1) === "2") {
-          app.flair += "2";
-        }
-        if (css_class && css_class.indexOf(' ') > -1) {
-          if (app.flair.indexOf('ribbon') > -1) {
-            css_class = css_class.substr(0, css_class.indexOf(' ')) + " " + app.flair;
-          } else {
-            css_class = app.flair + " " + css_class.substr(css_class.indexOf(' ') + 1);
-          }
-        } else {
-          if (app.flair.indexOf('ribbon') > -1 && css_class && css_class.indexOf('ribbon') === -1) {
-            css_class = css_class + " " + app.flair;
-          } else if (app.flair.indexOf('ribbon') === -1 && css_class && css_class.indexOf('ribbon') > -1) {
-            css_class = app.flair + " " + css_class;
-          } else {
-            css_class = app.flair;
-          }
-        }
-        Reddit.setUserFlair(req.user.redToken, user.name, css_class, flair, app.sub).then(function () {
-          Event.create({
-            type: "flairTextChange",
-            user: req.user.name,
-            content: "Changed " + user.name + "'s flair to " + css_class
-          }).exec(function () {
-          });
-
-          console.log("/u/" + req.user.name + ": Changed " + user.name + "'s flair to " + css_class);
-          Reddit.sendPrivateMessage(
-            refreshToken,
-            'FlairHQ Notification',
-            'Your application for ' + formatted + ' flair on /r/' + app.sub + ' has been approved.',
-            user.name).then(undefined, function () {
-              console.log('Failed to send a confirmation PM to ' + user.name);
-            }
-          );
-          if (app.sub === 'pokemontrades') {
-            user.flair.ptrades.flair_css_class = css_class;
-          } else {
-            user.flair.svex.flair_css_class = css_class;
-          }
-          user.save(function (err) {
-            if (err) {
-              console.log(err);
-            }
-          });
-          Application.destroy({id: req.allParams().id}).exec(function (err, app) {
-            if (err) {
-              return res.serverError(err);
-            }
-            return res.ok(app);
-          });
-        }, function (err) {
-          return res.serverError(err);
-        });
-      });
-    });
+      var user = await User.findOne(app.user),
+        shortened = app.sub === 'pokemontrades' ? 'ptrades' : 'svex';
```

Comment:
Can you remove the comma from this and just define it seperately. Mainly because the await makes it confusing to look at with that.